src/ds/ts/add-mod-del.md

```diff
@@ -0,0 +1,66 @@
+## Manage data points
+
+To add new samples to a time series, use the `TS.ADD` and `TS.MADD` commands. Note that if the series does not exist, it will be automatically created.
```

Comment:
`TS.MADD` does not create a new time series. Only `TS.ADD`.

Reply:
I think it is worth mentioning here that 
- Each sample is a {time tag, value} pair
- Time tags are measured in milliseconds since the Unix Epoch (January 1st, 1970, at 00:00:00 UTC)
- Time tags can be specified by the client or filled automatically by the server.
- Values are 64-bit floating-points